database/seedData.js

```diff
@@ -79,18 +73,15 @@ const sharedProduct5 = new ReviewSummary({
   reviews: [], // 2
 });
 
-sharedProduct1.save((err) => {
-  if (err) {
-    console.error(err);
-    return;
-  }
-  console.log('successfully saved product 1');
-});
-
-sharedProduct3.save((err) => {
-  if (err) {
-    console.error(err);
-    return;
-  }
-  console.log('successfully saved product 3');
-});
+const savePromises = [
+  sharedProduct1.save(),
+  sharedProduct2.save(),
+  sharedProduct3.save(),
+  sharedProduct4.save(),
+  sharedProduct5.save(),
+];
+
+db.dropDatabase()
```

Comment:
Does this database need to be dropped? It seems that the database is being seeded and then immediately dropped afterwards.

Reply:
The purpose of this is to drop whatever still exists in the DB before saving new data. Drop first, then seed.